scripts/prepare_data.py

```diff
@@ -51,6 +137,22 @@ def add_tile_id(row):
     # Load input parameters
     OUTPUT_DIR = cfg['output_folder']
     SHPFILE = cfg['datasets']['shapefile']
+    CATEGORY = cfg['datasets']['category'] if 'category' in cfg['datasets'].keys() else False
+    FP_SHPFILE = cfg['datasets']['fp_shapefile'] if 'fp_shapefile' in cfg['datasets'].keys() else None
+    EPT_YEAR = cfg['datasets']['empty_tiles_year'] if 'empty_tiles_year' in cfg['datasets'].keys() else None
+    if 'empty_tiles_aoi' in cfg['datasets'].keys() and 'empty_tiles_shp' in cfg['datasets'].keys():
+        logger.error("Choose between supplying an AoI shapefile ('empty_tiles_aoi') in which empty tiles will be selected, or a shapefile with selected empty tiles ('empty_tiles_shp')")
+        sys.exit(1)    
+    elif 'empty_tiles_aoi' in cfg['datasets'].keys():
+        EPT_SHPFILE = cfg['datasets']['empty_tiles_aoi']
+        EPT = 'aoi'
+    elif 'empty_tiles_shp' in cfg['datasets'].keys():
+        EPT_SHPFILE = cfg['datasets']['empty_tiles_shp'] 
+        EPT = 'shp'
+    else:
+        EPT_SHPFILE = None
+        EPT = None
+    CATEGORY = cfg['datasets']['category'] if 'category' in cfg['datasets'].keys() else False
     ZOOM_LEVEL = cfg['zoom_level']
 
     # Create an output directory in case it doesn't exist
```

Comment:
The two next lines can be replaced with `os.makedirs(OUTPUT_dir, exists_ok=True)`